src/main/cpp/subsystems/Arm.cpp

```diff
@@ -0,0 +1,14 @@
+// #include "subsystems/Arm.h"
+
+// Arm::Arm() {
```

Comment:
You need to keep the constructor and other methods uncommented, even if there is nothing inside the function